R/dvhydrograph-render.R

```diff
@@ -287,6 +300,11 @@ getDVHydrographPlotConfig <- function(plotItem, plotItemName, yLabel="", minMaxE
                            lty=ifelse(plotItem[['newSet']] == "est", 1, 6), col=ifelse(plotItem[['newSet']] == "est", "orange", "red3")),
                       styles$est_lines)
     ),
+    estimated4Edges = list(
+      arrows = append(list(x0=plotItem[['time']], x1=plotItem[['time']], y0=plotItem[['y0']], y1=plotItem[['y1']], 
+                           lty=ifelse(plotItem[['newSet']] == "est", 1, 7), col=ifelse(plotItem[['newSet']] == "est", "black", "red3")),
```

Comment:
Yeah, `ifelse` only likes returning one value.

This returns the first list item, so `list(2)` not `list(2,2,2)`.
```
ifelse(1==1, list(2,2,2), list(3,3,3))
```